azurelinuxagent/ga/update.py

```diff
@@ -199,7 +199,10 @@ def run_latest(self, child_args=None):
         if self.signal_handler is None:
             self.signal_handler = signal.signal(signal.SIGTERM, self.forward_signal)
 
-        latest_agent = None if not conf.get_autoupdate_enabled() else self.get_latest_agent_greater_than_daemon(
+        # If new flag explicitly set, agent will use latest agent downloaded and will not fall back to installed agent. See the new flag definition in conf.py
+        use_latest_agent = conf.is_present("AutoUpdate.UpdateToLatestVersion") or conf.get_autoupdate_enabled()
```

Comment:
needs to be present and be True

Reply:
No- for preventing further new agent downloads

Reply:
Once agent is on particular version, AutoUpdate.UpdateToLatestVersion=n should not revert to installed version.

Reply:
yes, value does not matter

we should add a warning message here as well... if autoupdate.enabled is also present, log/telemetry that it will be ignored